pkgs/top-level/all-packages.nix

```diff
@@ -24469,6 +24471,8 @@ in
     wineRelease = "staging";
   });
 
+  vkd3d = callPackage ../misc/emulators/wine/vkd3d.nix {};
```

Comment:
Just a `let vkd3d = callPackage ./vkd3d.nix {}` inside base.nix, it not used anywhere outside wine (and probably never be). (not sure about faudio)

Reply:
Faudio is used (and made for) for FNA, a reimplementation of XNA.